modules/globebrowsing/globes/renderableglobe.h

```diff
@@ -74,9 +80,23 @@ class RenderableGlobe : public Renderable {
         properties::BoolProperty atmosphereEnabled;
         properties::BoolProperty useAccurateNormals;
         properties::FloatProperty lodScaleFactor;
-        properties::FloatProperty cameraMinHeight;
+        properties::FloatProperty cameraMinHeight;        
         properties::FloatProperty orenNayarRoughness;
     };
+
+    // Shadow structure
+    typedef struct {
```

Comment:
Where? If you are talking about the same structure in renderableplanet, remember: it will be removed later. Inside renderableatmosphere it is needed to implement the eclipse shadow from inside the atm perspective. If someone wants to have an eclipse with no atm, it is still possible. But I agree about the duplicated code. I will uncouple this code.